.changeset/four-ties-love.md

```diff
@@ -2,4 +2,4 @@
 '@hono/qwik-city': patch
 ---
 
-#94 81fe019: update qwic
+#94 81fe019: update Qwic
```

Comment:
@totto2727 

Hey again and again. It should be "Qwi**k**"!

<img width="176" alt="Screenshot 2023-06-22 at 22 35 19" src="https://github.com/honojs/middleware/assets/10682/b274ad23-42ea-406b-97d1-b671b075191b">

Reply:
Sorry…
I forgot to git add…